### PR TITLE
0.1.7

### PR DESCRIPTION
**Pull request by zhady009**
for intellij 2019.2 and clean some code

### CODE REVIEW

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -149,7 +152,7 @@
     <completion.contributor language="nginx" implementationClass="net.ishchenko.idea.nginx.NginxCompletionContributor"/>
     <gotoFileContributor implementation="net.ishchenko.idea.nginx.NginxChooseByNameContributor"/>
     <programRunner implementation="net.ishchenko.idea.nginx.run.NginxRunner"/>
-    <fileTypeFactory implementation="net.ishchenko.idea.nginx.NginxFileTypeFactory"/>
+    <fileType name="nainx file" implementationClass="net.ishchenko.idea.nginx.NginxFileType" language="nginx" extensions="nginx" fieldName="INSTANCE" />
```
Comment (Deadleg) [inline resources/META-INF/plugin.xml]: Minor typo `nainx`

File: src/net/ishchenko/idea/nginx/KeywordsFromSourcesGenerator.java [hunk 2]
```diff
@@ -31,11 +31,11 @@
 
     private static final Pattern PREPROCESSOR_DIRECTIVE_PATTERN = Pattern.compile("#.+");
 
-    private static final Pattern DIRECTIVE_BLOCK_PATTERN = Pattern.compile("static\\s+ngx_command_t\\s+[^\\[]+\\[\\]\\s*=\\s*(\\{[^;]+);");
```
Comment (Deadleg) [inline src/net/ishchenko/idea/nginx/KeywordsFromSourcesGenerator.java]: I'm not familiar with this regex, can you explain what this fixes?

**Comment by Deadleg** [comment]
Thanks for fixes and cleaning the code at the same!

File: src/net/ishchenko/idea/nginx/KeywordsFromSourcesGenerator.java (hunk 2 above)
Comment (zhady009) [reply src/net/ishchenko/idea/nginx/KeywordsFromSourcesGenerator.java]: It's redundant character escape '\\]' in regexp and it has the same function as before.

File: resources/META-INF/plugin.xml (hunk 1 above)
Comment (zhady009) [reply resources/META-INF/plugin.xml]: already fixed. thanks

**Pull request merged**